Formula/libcouchbase.rb

```diff
@@ -1,16 +1,10 @@
 class Libcouchbase < Formula
   desc "C library for Couchbase"
-  homepage "https://docs.couchbase.com/c-sdk/2.10/start-using-sdk.html"
-  url "https://packages.couchbase.com/clients/c/libcouchbase-2.10.6.tar.gz"
-  sha256 "341d7af60d637c81a4de623168746b699f49a75358aa33d9b1b99dd0a63a7cf3"
+  homepage "https://docs.couchbase.com/c-sdk/3.0/hello-world/start-using-sdk.html"
```

Comment:
```suggestion
  homepage "https://docs.couchbase.com/c-sdk/3.0/"
```

Reply:
@SMillerDev That URI doesn't work and takes you to the home page rather than the libcouchbase-specific page.  I've made the homepage as specific as possible.